sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -215,7 +208,9 @@ function processFormData(formData?: FormDataMap) {
   for (const element in formData) {
     const item = formData[element];
     if (item instanceof Uint8Array) {
-      processedFormData[element] = binaryArrayToString(item);
+      // Some RLCs take a Uint8Array for the parameter, whereas FormDataMap expects
+      // a File or a Blob, so we need to wrap it.
+      processedFormData[element] = new Blob([item]);
     } else {
       processedFormData[element] = item;
```

Comment:
Maybe not relevant to this change, just curious if we support json object in form data policy? https://github.com/Azure/cadl-ranch/pull/460/files#diff-6cd2a3081b7878b35614372bc58ebd160b688d8bc3811f4eb6797ff795829722R21

Reply:
At the moment if you want to send JSON as a part in a multipart request it is the caller's responsibility to encode the JSON and wrap it in a `Blob` and set the content type appropriately.

If we ever get an example of JSON in a multipart body like this, it would be an additive change to support, but I am hesitant to do so unless we have a real-world example.

Reply:
We have a real case in FaceAI where they need to pass both bytes and json objects.

https://github.com/Azure/azure-rest-api-specs-pr/blob/887741d7b74b601350d58904a723609772ff51c9/specification/ai/Face/Sessions/models.tsp#L166-L174.